docs/source/quickstart-docker.rst

```diff
@@ -1,11 +1,17 @@
 Install JupyterHub with Docker
 ==============================
 
+
+**important:**
```

Comment:
This is now duplicated.

This same message is already highlighed via the `.. important::` below:

<img width="710" alt="Screen Shot 2022-10-21 at 10 20 01" src="https://user-images.githubusercontent.com/151929/197148556-bf820410-6eea-4556-ad49-87ae2ac5094d.png">